src/app/server/Server.h

```diff
@@ -212,6 +212,10 @@ struct ServerInitParams
     // for ember/zap-generated models.
     chip::app::DataModel::Provider * dataModelProvider = nullptr;
 
+    // Product could set customzied ProviderChangeListener.
```

Comment:
Do not use the word "product".

```suggestion
    // Implementers can customize the ProviderChangeListener to possibly synchronize/delay/aggregate reports.
```